### PR TITLE
Add new guide and replace links

### PR DESCRIPTION
**Pull request by wrightmartin**
Minor text change

### CODE REVIEW

File: ainow/templates/ainow/base.html [hunk 1]
```diff
@@ -110,7 +110,7 @@ <h2 class="mysoc-footer__site-name">TICTeC</h2>
                 <nav class="mysoc-footer__links">
                     <ul>
                         <li role="presentation"><a href="/">About</a></li>
-                        <li role="presentation"><a href="https://www.mysociety.org/files/2018/12/tictec-guide.pdf">Guide to TICTeC</a></li>
+                        <li role="presentation"><a href="{% static 'guide-2019.pdf' %}>Guide to TICTeC</a></li>
```
Comment (chrismytton) [inline ainow/templates/ainow/base.html]: This is missing a `"` after the `%}`, which means the link isn't actually appearing in the footer!

**Review by chrismytton — COMMENTED** [review]
One tiny problem, but I'll push a fix for that now and then get this deployed.

**Comment by chrismytton** [comment]
This is now live ➡️ https://tictec.mysociety.org/

**Comment by wrightmartin** [comment]
thanks @chrismytton!

**Pull request merged**